src/main/java/mate.academy/service/ProductParser.java

```diff
@@ -1,7 +1,9 @@
 package mate.academy.service;
 
+import mate.academy.lib.Component;
 import mate.academy.model.Product;
 
+@Component
```

Comment:
there's no point on annotating interfaces as classes that are used for instance creation, because we cannot create object from interface